pallets/moonbeam-xcm-benchmarks/src/generic/benchmarking.rs

```diff
@@ -41,8 +41,36 @@ benchmarks! {
 
 	} : {
 		executor.bench_process(xcm)?;
-	} verify {
+	}
 
+	exchange_asset {
+	} : {
+		BenchmarkResult::from_weight(T::BlockWeights::get().max_block)
```

Comment:
Will this work? Substrate is using a slightly different approach of wrapping this result in the error and returning it. https://github.com/paritytech/substrate/blob/master/frame/elections-phragmen/src/benchmarking.rs#L334